### ISSUE TITLE
Ping/Pong example bot not connecting to Discord from Windows.

### ISSUE THREAD

**Issue by predatorian3** [bwmarrin/discordgo#619]
I am unable to connect to Discord with my Bot token using discordgo. I `go get`'ed the library and typed out the ping/pong example. However, it didn't seem to connect. So I copied and pasted the example, and same result. I then tried again with the discordrb framework and my token, and it was able to connect. 

```golang
package main

import (
	"github.com/bwmarrin/discordgo"
	"log"
	"os"
	"os/signal"
	"syscall"
)

var (
	Token string
)

func init() {
	Token = os.Getenv("DGU_TOKEN")
}

func main() {
	dg, err := discordgo.New("Bot " + Token)
	if err != nil {
		log.Fatal("Error creating Discord Session,", err)
	}

	dg.AddHandler(messageCreate)

	err = dg.Open()
	if err != nil {
		log.Fatal("Error opening connection,", err)
	}

	log.Println("Bot is now running. Press CTRL+c to exit")
	sc := make(chan os.Signal, 1)
	signal.Notify(sc, syscall.SIGINT, syscall.SIGTERM, os.Interrupt, os.Kill)
	<-sc

	dg.Close()
}

func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
	if m.Author.ID == s.State.User.ID {
		return
	}
	if m.Content == "ping" {
		s.ChannelMessageSend(m.ChannelID, "Pong!")
	}
	if m.Content == "pong" {
		s.ChannelMessageSend(m.ChannelID, "Ping!")
	}
}
```

No errors presented from the binary however. 

Operating System information:

```
C:\Users\Phillip\Desktop\discord_bot_playground (master -> origin)
λ go build ping_pong_example_1.go

C:\Users\Phillip\Desktop\discord_bot_playground (master -> origin)
λ ping_pong_example_1.exe
2018/12/02 14:25:09 Bot is now running. Press CTRL+c to exit

C:\Users\Phillip\Desktop\discord_bot_playground (master -> origin)
λ systeminfo.exe

Host Name:                 FENIX
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.17134 N/A Build 17134
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
```

**Comment by iopred** [comment]
Do you happen to be using an old version? The older master (read: a month
ago) had an outdated API version.

**Comment by predatorian3** [comment]
I think this is what you're asking? 

```
C:\Users\Phillip\go\src\github.com\bwmarrin\discordgo (master -> origin)
λ git log
commit d420e28024ad527390b43aa7f64e029083e11989 (HEAD -> master, origin/master, origin/HEAD)
Author: Anmol Sethi <nhooyr@users.noreply.github.com>
Date:   Sat Jul 8 11:09:04 2017 -0400

    voice.go: remove runtime.LockOSThread call from opusSender (#404)

    runtime.LockOSThread does not do anything for performance and is instead
    intended for interfacing with libraries that use thread local data structures.

    Closes #402 
```

You're right. I just needed to `go get -u github.com/bwmarrin/discordgo`

```
C:\Users\Phillip\go\src\github.com\bwmarrin\discordgo (master -> origin)
λ git log
commit 8325a6bf6dd6c91ed4040a1617b07287b8fb0eba (HEAD -> master, tag: v0.19.0, origin/master, origin/HEAD)
Merge: efc0516 94c6dec
Author: Chris Rhodes <iopred@gmail.com>
Date:   Fri Nov 2 11:20:04 2018 -0700

    Merge branch 'develop'
```

I had totally forgot that doing just a plain `go get` doesn't update your local copy if you have one already.
